jenkins/release.jenkinsFile

```diff
@@ -82,6 +82,150 @@ pipeline {
                             mavenArtifactsPath: "$WORKSPACE/repository/",
                             autoPublish: true
                         )
+
+                        def dockerCopyHubConsole =
```

Comment:
What do you think of this refactor [1] to make it easier to confirm what is being pushed where? 

- [1] https://github.com/peterzhuamazon/opensearch-migrations/pull/1

Reply:
Thanks @peternied.

Reply:
Approved.